projects/admin-core/src/lib/application/application-layer-settings/application-layer-settings.component.ts

```diff
@@ -336,4 +346,23 @@ export class ApplicationLayerSettingsComponent implements OnInit, OnDestroy {
     }
   }
 
+  private getCrsText(serviceLayer: ExtendedGeoServiceAndLayerModel | null): string {
+    if (!serviceLayer) {
+      return '';
+    }
+    const crs: string[] = [];
+    const layersInService: ExtendedGeoServiceLayerModel[] = [];
+    this.store$.select(selectGeoServiceLayersByGeoServiceId(serviceLayer?.service.id))
```

Comment:
This is not how RxJS should be used. You should return an Observable and you can map the selector to a list of crs strings